.github/CODEOWNERS

```diff
@@ -90,6 +90,7 @@
 # Docker
 /docker/ @iotaledger/infrastructure @iotaledger/node @iotaledger/devops-admin
 /dev-tools/ @iotaledger/infrastructure @iotaledger/node @iotaledger/devops-admin
+/dev-tools/iota-indexer-monitoring/ @iotaledger/infrastructure
```

Comment:
Maybe your team can push the changes from https://github.com/iotaledger/iota/issues/6371 directly in this PR as well, so you don't need to wait for the review on this twice.

Reply:
As we will need to request different reviewers should be the same in terms of operations. So let's keep it separate.